float32 base_reading
---
float32 count
---
float32 count
